{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--025764b5-9667-4121-8bd1-9a511926543e",
    "created": "2024-08-13T16:04:20.673312Z",
    "modified": "2024-08-13T16:04:20.673317Z",
    "name": "No Title Available",
    "description": "In the Linux kernel before 4.14, an out of boundary access happened in drivers/nvme/target/fc.c.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2017-18379"
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://github.com/torvalds/linux/commit/0c319d3a144d4b8f1ea2047fd614d2149b68f889"
        },
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=0c319d3a144d4b8f1ea2047fd614d2149b68f889"
        },
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "https://support.f5.com/csp/article/K74012105"
        },
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "https://support.f5.com/csp/article/K74012105?utm_source=f5support&amp%3Butm_medium=RSS"
        }
    ]
}